onnx/test/large_onnx_test.py

```diff
@@ -0,0 +1,133 @@
+    Y = onnx.helper.make_tensor_value_info("Y", onnx.TensorProto.FLOAT, [None])
+    graph = onnx.helper.make_graph(
+        [
+            onnx.helper.make_node("MatMul", ["X", "A"], ["XA"]),
+            onnx.helper.make_node("MatMul", ["XA", "B"], ["XB"]),
+            onnx.helper.make_node("MatMul", ["XB", "C"], ["Y"]),
+        ],
+        "mm",
+        [X],
+        [Y],
+        [
+            onnx.numpy_helper.from_array(
+                np.arange(9).astype(np.float32).reshape((-1, 3)), name="A"
+            ),
+            onnx.numpy_helper.from_array(
+                (np.arange(9) * 10).astype(np.float32).reshape((-1, 3)),
+                name="B",
+            ),
+            onnx.numpy_helper.from_array(
+                (np.arange(9) * 10).astype(np.float32).reshape((-1, 3)),
+                name="C",
+            ),
+        ],
+    )
+    onnx_model = onnx.helper.make_model(graph)
+    onnx.checker.check_model(onnx_model)
+    return onnx_model
+
+
+def _large_linear_regression():
+    X = onnx.helper.make_tensor_value_info("X", onnx.TensorProto.FLOAT, [None, None])
+    Y = onnx.helper.make_tensor_value_info("Y", onnx.TensorProto.FLOAT, [None])
+    graph = onnx.helper.make_graph(
+        [
+            onnx.helper.make_node("MatMul", ["X", "A"], ["XA"]),
+            onnx.helper.make_node("MatMul", ["XA", "B"], ["XB"]),
+            onnx.helper.make_node("MatMul", ["XB", "C"], ["Y"]),
+        ],
+        "mm",
+        [X],
+        [Y],
+        [
+            onnx.large_helper.make_large_tensor_proto(
+                "#loc0", "A", onnx.TensorProto.FLOAT, (3, 3)
+            ),
+            onnx.numpy_helper.from_array(
+                np.arange(9).astype(np.float32).reshape((-1, 3)), name="B"
+            ),
+            onnx.large_helper.make_large_tensor_proto(
+                "#loc1", "C", onnx.TensorProto.FLOAT, (3, 3)
+            ),
+        ],
+    )
+    onnx_model = onnx.helper.make_model(graph)
+    large_model = onnx.large_helper.make_large_model(
+        onnx_model.graph,
+        {
+            "#loc0": (np.arange(9) * 100).astype(np.float32).reshape((-1, 3)),
+            "#loc1": (np.arange(9) + 10).astype(np.float32).reshape((-1, 3)),
+        },
+    )
+    large_model.check_model()
+    return large_model
+
+
+class TestLargeOnnx(unittest.TestCase):
+    def test_large_onnx_no_large_initializer(self):
+        model_proto = _linear_regression()
+        assert isinstance(model_proto, onnx.ModelProto)
+        large_model = onnx.large_helper.make_large_model(model_proto.graph)
+        assert isinstance(large_model, onnx.large_helper.ModelContainer)
+        with tempfile.TemporaryDirectory() as temp:
+            filename = os.path.join(temp, "model.onnx")
+            large_model.save(filename)
+            copy = onnx.large_helper.ModelContainer()
+            with self.assertRaises(RuntimeError):
+                assert copy.model_proto
```

Comment:
```suggestion
               copy.model_proto
```

was this the intention?

Reply:
Yes, model_proto can exist and be None